Doberman/Reading.py

```diff
@@ -115,6 +116,22 @@ def more_processing(self, value):
             value = int(value)
         return value
 
+    def sensiblesigfigs(self, reading, lowlim, upplim, defaultsigfigs=3):
```

Comment:
Does this need to be a member function? It doesn't use `self` so I think this can get moved to the module scope rather than class scope.

Reply:
Changed